src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -848,7 +848,10 @@ class CreateVmModal extends React.Component {
             if (value && value.profiles)
                 stateDelta.profile = value.profiles.sort().reverse()[0];
 
-            if (value && value.minimumResources.ram) {
+            // Hyperbola and some version of alpinelinux have incorrect minimum memory
+            // https://gitlab.com/libosinfo/osinfo-db/-/issues/95
+            if (value && value.minimumResources.ram &&
+                !["alpinelinux3.15", "alpinelinux3.15", "hyperbola03"].includes(value.shortId)) {
```

Comment:
What does it use instead? It seems to me that with the rounding fix this should be sufficiently unbroken. The user can still tweak it, and once it gets fixed in libosinfo, we don't have to revert the hack (especially as nothing will tell us when it gets fixed).

Reply:
I added it here because now when its broken in libosinfo, even with rounding fix, it will provide user with incorrect value for minimum resource and it would default to an incorrect value (only rounded up)
In this screenshot, the correct value according to alpine linux docs should be 128MiB, but because of a typo in libosinfo, it says 137MiB:
![Screenshot from 2022-04-25 13-17-27](https://user-images.githubusercontent.com/42733240/165078855-b6f4539b-ecc5-4299-95a2-a3f2e7c3295f.png)
And I think for alpine linux it's better to show no value then incorrect one:
![Screenshot from 2022-04-25 13-21-19](https://user-images.githubusercontent.com/42733240/165079342-399df167-6f3b-412d-b8c8-95ee9ab2fa9a.png)

Reply:
However it's true that nothing would tell us when it gets fixed. Maybe we can add a assert in test which will warn us in the future?

Reply:
I still think that proposing 137 MB is not so bad that we have to work around it. It's not wildly off, and the user can still change it. Of course fixing the rounding is important.

Reply:
Alright, dropped the workaround commit for alpine linux, now there is just rounding fix and and fix for the case when storage and memory requirements are not known